SR2025/2025-04-24-competition-complete.md

```diff
@@ -0,0 +1,66 @@
+---
+to: Student Robotics 2025
+subject: Student Robotics 2025 comes to a close
+---
+
+![Competition Photo](https://studentrobotics.org/images/content/blog/sr2025/sr2025-photo.jpg TODO_IMAGE)
+
+Hi everyone,
+
+We'd like to thank you all for taking part in Student Robotics 2025 and making it such a successful event! An article about the competition, results, and game has been [posted on our website](https://studentrobotics.org/blog/2025-04-23-ths-win-sr2025/ TODO_BLOG_POST).
+
+Congratulations again to our winners:
+
+| Prize                           | Team
+|---------------------------------|--------------------------------
+| 1st Place                       | The Highfield School
+| 2nd Place                       | School of St Helen and St Katharine
+| 3rd Place                       | Ruthin School
+| Rookie Award                    | The Highfield School
+| Excellence in Engineering Award | Queen Mary’s College
+| Robot and Team Image            | School of St Helen and St Katharine
+| Online Presence                 | School of St Helen and St Katharine
+
+## Event Media
+
+If you'd like some mementos of the competition or would just like to see the photos which were taken, we've collected them all together into a photo album, which will be added to the blog post in the coming weeks.
+
+The livestreams of the matches are also available on YouTube, including footage from the final:
+
+* Day 1: https://www.youtube.com/watch?v=iJwuYluRUjE
+* Day 2: https://www.youtube.com/watch?v=m4DT0_MfAuk
+
+The in-venue soundtrack playlists can also be found on [the event page](https://studentrobotics.org/events/sr2025/competition/#soundtrack).
+
+## Feedback
+
+We'd love to get feedback on how the competition was for you and your competitors this year, so we know what we did well, where we can improve, and to help apply for future funding.
+
+Please share the below short, anonymous survey with your competitors: [Send us feedback](https://forms.gle/J1c5z8uzmHnxvGGK8)
+
+Additionally, we welcome feedback from you the supervisors too. Please fill in this survey: [Supervisor Feedback](https://forms.gle/Z4oKARtXfdrPg6kX7)
+
+Without all of your invaluable feedback we wouldn't be able to run the competition and attract new sponsors.
+Please submit any feedback by 18th May.
```

Comment:
Question: 18th is quite a long time after the event. By that time, people may have forgotten much of the detail. Bringing it 1 week forward means the deadline is a month after the competition (and 2 weeks after this email) which feels more appropriate.

Reply:
Yeah, happy to do that. We haven't had a deadline in the past, so this was more about ensuring the results were in in time for when we wanted to look at them (Will noted that the kit team wanted to look at them on the 20th).